config/locales/en/layouts.yml

```diff
@@ -2,9 +2,9 @@
 en:
   layouts:
     phase_banner:
-      text: This is an MVP. Some things may not work as expected.
+      text_html: This is a new service – your <a href="%{survey_url}">feedback</a> will help us to improve it
```

Comment:
In the footer link we added the rel="external"
Do you want to do the same here?

Reply:
Good idea, thanks @zheileman. @MohammedHassanLatif93 Let's take a look at this after our break